spec/system/viewing_sidebar_mobile_spec.rb

```diff
@@ -5,7 +5,10 @@
   let(:sidebar_dropdown) { PageObjects::Components::SidebarHeaderDropdown.new }
   let(:composer) { PageObjects::Components::Composer.new }
 
-  before { sign_in(user) }
+  before do
+    SiteSetting.navigation_menu = "sidebar"
```

Comment:
Had to do this since locally you can have different site setting values saved in the test DB

Reply:
This likely mean that your test DB is polluted somehow because `navigation_menu` defaults to `"sidebar"`. can you try resetting your test DB locally?

Reply:
I find this happens a lot with the test DB locally, will give it a drop + create though and see whether that works.

Reply:
Okay that fixed it, reverting this part of the change 👍